src/main/java/com/amplitude/Constants.java

```diff
@@ -3,6 +3,13 @@
 public interface Constants {
 
   String API_URL = "https://api2.amplitude.com/2/httpapi";
+  String BATCH_API_URL = "https://api2.amplitude.com/batch";
+
+  enum httpCallMode {
```

Comment:
let's not put enum inside Constant class... if you want you can possibly leave in in this file, but move out of the Constant class. Also, you can just use the new enum class to manage the value of url.
also the enum class name should be `HttpCallMode`